packages.ini

```diff
@@ -23,6 +23,7 @@ php7.2 = bionic
 php7.2-curl = bionic
 php7.2-gd = bionic
 php7.2-pgsql = bionic
+php7.2-xml = bionic
```

Comment:
Where does these come from ?

Reply:
simplepie requires it, but has only recently added a check that fails if it doesn't exist - https://github.com/simplepie/simplepie/pull/590